bin/setup

```diff
@@ -18,3 +18,10 @@ fi
 mix ecto.create
 npm install yarn
 ./node_modules/.bin/yarn install
+
+if [ "$CI" ]; then
+  change-phantomjs-version 2.1.1
```

Comment:
This is latest available stable version with better stability, perfomance and fixed bugs. It should be specified manually because the default PhantomJS version on SemaphoreCI is `1.9.8`